src/lib/viewers/media/MediaControls.js

```diff
@@ -84,6 +83,8 @@ class MediaControls extends EventEmitter {
         this.handleAutoplay = this.handleAutoplay.bind(this);
         this.handleSubtitle = this.handleSubtitle.bind(this);
         this.handleAudioTrack = this.handleAudioTrack.bind(this);
+        this.handleFullscreenEnter = this.handleFullscreenEnter.bind(this);
```

Comment:
No changes were needed for the regular controls?

Reply:
I updated all locations where `fullscreen` was being used. It looks like the fullscreen button for all other viewers is added via `BaseViewer.js`.